README.md

```diff
@@ -1,54 +1,316 @@
+[instructions for dynamic websites](#231-for-dynamic-websites). If your Web
+pages are pure HTML, please follow the
+[instructions for static websites](#232-for-static-websites).
+
+#### 2.3.1. For dynamic websites
+When your Web pages are generated by PHP files, you need to require WOVN.php
+interception script within each PHP file generating content. Please use the
+following code. It must be inserted at the beginning of PHP files.
+```
+require_once('/website/root/directory/WOVN.php/src/wovn_interceptor.php');
+```
+
+#### 2.3.2. For static websites
+When your Web pages are pure HTML, you need to create a `wovn_index.php` file
+that you will use to serve and localize your HTML pages. We recommend you to
+start with the sample that we provide.
 ```
-root_directory/wovn.ini
-root_directory/WOVN.php/wovn.ini.sample
+$ cp WOVN.php/wovn_index_sample.php wovn_index.php
+```
+**Note for SSI users:** if you are using our sample `wovn_index.php`, please
+follow the `# SSI USER` instructions inside the code.
+
+Once your `wovn_index.php` is setup, you need to configure your website so that
+any request to HTML pages are redirected to `wovn_index.php`. If you are using
+an Apache server, please follow the
+[instructions for Apache](#redirect-to-wovn_indexphp-with-apache). If you are
+using Nginx (without Apache), please follow the
+[instructions for Nginx](#redirect-to-wovn_indexphp-with-nginx).
+
+#### Redirect to `wovn_index.php` with Apache
+For redirecting requests to `wovn_index.php`, we recommend using `.htaccess`
+configuration with `mod_rewrite` PHP module. Please follow the
+[official instructions](https://httpd.apache.org/docs/2.4/) for installing and
+activating `mod_rewrite` module (in some cases, `mod_rewrite` is already
+installed but not activated).
+
+Bellow is the `.htaccess` configuration you should use.
+```
+<IfModule mod_rewrite.c>
+  RewriteEngine On
+
+  # Don't intercept .cgi files, as they won't execute
+  RewriteCond %{THE_REQUEST} \.cgi
+  RewriteRule .? - [L]
+
+  # Intercept only static content: html and htm urls
+  # Warning: do not remove this line or other content could be loaded
+  RewriteCond %{REQUEST_URI} /$ [OR]
+  RewriteCond %{REQUEST_URI} \.(html|htm|shtml|php|php3|phtml)
+  # Use the wovn_index.php to handle static pages
+  RewriteRule .? wovn_index.php [L]
+</IfModule>
+```
+
+Alternatively, you can also copy the file `htaccess_sample` from `WOVN.php`
+directory.
+```
+$ cp WOVN.php/htaccess_sample .htaccess
+```
+
+#### Redirect to `wovn_index.php` with Nginx
+For redirecting to `wovn_index.php`, you need to update your Nginx configuration
+(`/etc/nginx/conf.d/site.conf`). Below is an highlight of the code you need to
+add in the configuration file.
+```
+server {
+  # ...
+
+  # php configuration
+  location ~ \.php$ {
+    # ...
+  }
+
+  location / {
+    # ...
+
+    # WOVN.php interception ####################################################
```

Comment:
This line seems unnecessary.

Reply:
Because Their configuration may contain a lot of unrelated stuffs, we usually put separating line to identify the WOVN.php part. It is unnecessary, but it help when the file is huge.

Reply:
It just seemed strange to include the organizing comments in the configuration instructions. But if it will help customers who simply copy-paste this content, I'm fine with keeping it in also.